pages/about-us/de.text

```diff
@@ -0,0 +1,94 @@
+@title = 'Über uns'
+
+h2. Warum Riseup gebraucht wird
+
+!>img/page22-1003-full_medium.jpg!
+
+Können wir uns auf kommerzielle Emailanbieter für vertrauliche Emailkommunikation verlassen? Nicht nur dass sie regelmäßig unsere Emails nach bestimmten Inhalten scannen, sie folgen außerdem den Forderungen von Regierungen, die digitale Freiheit einzuschränken ohne klare Prinzipien zur Gewährleistung der Privatssphäre ihrer Nutzer*innen. Nicht zu reden von offensichtlichen kommerziellen Interessen der Unternehmen, die unseren Ansprüchen zuwiderlaufen. Die US-Regierung praktiziert umfassende Überwachung (“full pipe monitoring”) und soziale Netzwerkanalyse auf allen Kommunikationskanälen, wodurch sie in die Lage versetzt weden, eine detaillierte Karte sozialer Bewegungen und ihrer Organisationsweise zu erstellen. Darüber hinaus gibt es ihnen Informationen, welche Verbindungen zerstört werden müssen, um soziale Bewegungen zu stören.
+
+Wir glauben, es ist lebenswichtig, dass essentielle Kommunikationsinfrastruktur durch Organisationen der Bewegungen und nicht Unternehmen oder Regierungen kontrolliert werden sollten.
+
+Wir streben an, Email so sicher und privat wie möglich zu haben. Wir speichern keine IP-Adressen. (Die meisten Dienste speichern detaillierte Verbindungsdaten in ihren Logdateien. Wir speichern nur Informationen, mit denen sich unsere Nutzer*innen nicht identifzieren lassen.). Alle unsere Daten, Email eingenommen, werden verschlüsselt gespeichert. Wir arbeiten hart daran, unsere Server sicher und gut geschützt gegenüber böswilligen Angriffen zu halten. Wir teilen keine Nutzer*innendaten mit irgendjemandem. Wir kämpfen aktiv gegen alle Versuche, uns mittels subpoena oder anderweitig zur Herausgabe von Nutzer*inneninformationen oder Logdateien zu zwingen. Wir werden ein- oder ausgehende Emails nicht lesen, durchsuchen, oder verarbeiten außer um euch mittels automatischer Skripte vor Viren und Spam zu schützen, oder wenn wir durch euch zur Fehlersuche dazu aufgefordert werden.
+
+h2.  Riseups Ziele
+
+!>img/riseup-yellow.gif!
+
+Das Riseup-Kollektiv ist eine autonome Körperschaft in Seattle mit kollektiven Mitgliedern weltweit. Unser Ziel ist es, Hilfe bei der Erschaffung einer freien Gesellschaft zu leisten, eine Welt mit Freiheit von Gier und Meinungsfreiheit, eine Welt ohne Unterdrückung oder Hierarchie, in der Macht gleichmäßig verteilt ist. Unser Beitrag ist das Sicherstellen von Kommunikations- und Computerresourcen für Alliierte in Kämpfen gegen Kapitalismus und andere Unterdrückungsformen.
```

Comment:
collective members meint hier vermutlich "Kollektivmitgliedern" or "Mitgliedern des Kollektivs".  Ich würde das Kollektiv da weglassen und schreiben "mit Mitgliedern auf der ganzen Welt".
rephrasing further:
"Unser Ziele sind, beim Aufbau einer freien Gesellschaft zu helfen, eine Welt ohne Gier und mit Meinungsfreiheit, eine Welt..."
Um dazu beizutragen, übernehmen wir Verantwortung für Kommunikations- und Computerresourcen für Alliierte in Kämpfen gegen Kapitalismus und andere Unterdrückungsformen .